dot/network/notifications.go

```diff
@@ -238,7 +243,12 @@ func (s *Service) sendData(peer peer.ID, hs Handshake, info *notificationsProtoc
 		return
 	}
 
-	if support, err := s.host.supportsProtocol(peer, info.protocolID); err != nil || !support {
+	support, err := s.host.supportsProtocol(peer, info.protocolID)
+	if err != nil {
+		logger.Errorf("could not check if protocol %s is supported by peer %s: %s", info.protocolID, peer, err)
+	}
+
+	if err != nil || !support {
```

Comment:
should we report a peer if some error occurs while trying to check if the protocol is supported by the peer? @noot @timwu20. I think if an error occurs we should just log and then return.

Reply:
@EclesioMeloJunior I believe this logic is from substrate - we can double check if substrate downscores a peer based on them not supporting a protocol

Reply:
that's a pretty good catch, thanks @EclesioMeloJunior